examples/search-index/searchIndexVector.json

```diff
@@ -0,0 +1,72 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
```

Comment:
@lantoli this looks correct, but would just double check against inputs / comments from Terraform one example- https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1605/files#r1391217936

Reply:
thanks, changing the example to match the other one

Reply:
SGTM!